source/enterprise/tech-guide/shopware-server-configuration-guide.md

```diff
@@ -5,59 +5,86 @@ github_link: enterprise/tech-guide/shopware-server-configuration-guide.md
+* Needs access to the */tmp* directory
+* Needs to own the directory directory where new shops can be installed
 
-This command should take you directly to a shell in your Shopware Server. If that doesn't happen, you need to review your SSH configuration. We recommend using SSH key authentication between your two servers. Additionally, for increased security, you should use different pairs of keys for connecting to different servers. You can find extensive documentation online about how to do this, and how to create a `.ssh/config` file that will reflect your desired configuration.
-  
-### Create a MySQL account
+## HowTo: Setup on Ubuntu 14.04
 
-Certain actions will require the Enterprise Dashboard to access your Shopware Server's database directly. For that reason, you should ensure that a `edb-deploy` MySQL user exists and has `edb-deploy` as a password. It must have permissions to create new databases, as well as access the existing ones. As a security measure, you can limit this user to local access. All MySQL actions from the Enterprise Dashboard are tunneled in SSH by `ansible`.
- 
- 
-### Handle file system permissions
+This HOWTO should help you understand the requirements postulated above, but be aware that based on your specific
+operating system and version the commands you actually have to execute may differ vastly. We assume that you
+have already setup a system that is capable of executing Shopware and has an configured apache host.
+
+First you create a MySQL user that has full access from the current host:
+
+````shell
+mysql -u _YOUR_USER_ -p _YOUR_PASSWORD_ -e "CREATE USER 'edb-deploy'@'localhost' IDENTIFIED BY 'edb-deploy'" 
+````
+> Notice: MySQL defaults to grant newly created users all rights. If you want to reduce this, please make sure that the user is still able to create tables.
 
-As a security precaution, we use file system ACLs, which should be active on your file system. Access to the `/tmp` folder is also required, for storing temporary data.
+
+Now you create the edb-deploy shell user and add him to the apache group (in this case `www-data`). 
+
+````shell
+useradd edb-deploy -s /bin/bash -m
+usermod -a -G edb-deploy www-data
+````
+> Notice: Although it is not necessary to specify a shell for the deployment user, I found it very helpful for debugging purposes.
+
+We need to add the Apache users group to the edb-deploy user
+
+````shell
+usermod -a -G www-data edb-deploy
+````
+
+After that we can securely set the ownership of the apache host directory to the deploy user, in this case simply `/var/www`
  
-### Configure your server on the Enterprise Dashboard
+````shell
+chown -R edb-deploy:edb-deploy /var/www
+````
+
+Now add the **public ssh key** of the Enterprise Dashboards background process user to the edb-deploy user. To enable SSH access from the Enterprise Dashboard.
+
+````shell
+touch /home/edb-deploy/.ssh/authorized_keys
+echo "PUBLIC KEY" >> /home/edb-deploy/.ssh/authorized_keys
```

Comment:
Dachte auch da müsste es ein Kommando geben, gefunden habe ich allerdings nichts - zumindest nichts für die lokale Maschine.... siehe: http://askubuntu.com/questions/46424/adding-ssh-keys-to-authorized-keys